tools/extra/pac/fpgabist/CMakeLists.txt

```diff
@@ -34,9 +34,12 @@ set(PYTHON_SRC
      bist_nlb3.py
      fpgabist)
 
- add_subdirectory(dma)
- add_subdirectory(dma_vc)
- add_subdirectory(bist)
+if(TBB_FOUND)
+  add_subdirectory(dma)
```

Comment:
An else clause here to print a message that DMA won't be built would save some debugging when it happens.

Thank you for resolving the TBB problem.

Reply:
For some reason, FindTbb.cmake isn't finding tbb/tbb.h.  I experimented with it a bit, but have instead made the condition if(TBB_LIBRARIES), which is being populated when tbb is present.  It's not elegant, but the alternative would be to always exclude the dma app.

Reply:
Is include missing because you have to check the tbb-devel package instead of tbb?

Reply:
I tried editing the pkg_check_modules call like so, but it still isn't found:
```cmake
pkg_check_modules(PC_TBB QUIET tbb-devel)
```